sympy/utilities/tests/test_lambdify.py

```diff
@@ -364,8 +364,11 @@ def test_numpy_old_matrix():
 def test_python_div_zero_issue_11306():
     if not numpy:
         skip("numpy not installed.")
-    p = Piecewise((1 / x, y < -1), (x, y <= 1), (1 / x, True))
-    lambdify([x, y], p, modules='numpy')(0, 1)
+    p = Piecewise((1 / x, y < -1), (x, y < 1), (1 / x, True))
+    f = lambdify([x, y], p, modules='numpy')
+    numpy.seterr(all='ignore')
```

Comment:
Yes, i guess so because the assert statement makes sure that the wrapper produces an output as `inf`.
Any specific cases you think it can give an error in ?